index.html

```diff
@@ -1,82 +1,327 @@
+                            <h1><a href="">Tonic</a></h1>
+                            <div class="work-info">
+                                <ul>
+                                    <li class="intro">Canopy</li>
+                                    <li> <i class="fa fa-circle"></i> Back End Dev</li>
+                                    <li><i class="fa fa-circle"></i> 2015</li>
+                                </ul>
+                            </div>
+                            <p>
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                            </p>
+
+                            <div class="tags">
+                                <ul>
+                                    <li>html</li>
+                                    <li>css</li>
+                                    <li>javascript</li>
+                                </ul>
+                            </div>
+
+                            <a href="#" class="btn-one">See project</a>
+                        </li>
+
+                    </ul>
+                </div>
+            </li>
+
+            <li class="work">
+                <div class="work-container">
+                    <ul class="work-pro">
+                        <li>
+                            <img src="./image/portfolio2.png" alt="Several hands holding beer glasses">
+                        </li>
+                        <li class="work-description">
+                            <h1><a href="">Multi-Post Stories</a></h1>
+                            <div class="work-info">
+                                <ul>
+                                    <li class="intro">FACEBOOK</li>
+                                    <li> <i class="fa fa-circle"></i> Full Stack Dev</li>
+                                    <li><i class="fa fa-circle"></i> 2015</li>
+                                </ul>
+                            </div>
+                            <p>
+                                Experimental content creation feature that allows users to add to an existing story over
+                                the course of a day without spamming their friends.
+                            </p>
+
+                            <div class="tags">
+                                <ul>
+                                    <li>html</li>
+                                    <li>css</li>
+                                    <li>javascript</li>
+                                </ul>
+                            </div>
+
+                            <a href="#" class="btn-one">See project</a>
+                        </li>
+                    </ul>
+                </div>
+            </li>
+            <li class="work">
+                <div class="work-container">
+                    <ul class="work-pro">
+                        <li>
+                            <img src="./image/portfolio3.png" alt="Several hands holding beer glasses">
```

Comment:
- [ ] Your image is not displayed, because your using the wrong path for your image, kindly fix this, by changing the path to ``` `"../Image/portfolio3.png"```